modules/text2vec-openai/vectorizer/class_settings.go

```diff
@@ -238,6 +255,13 @@ func (cv *classSettings) validateIndexState(class *models.Class, settings ClassS
 		"indexing.")
 }
 
+func (ic *classSettings) validateAzureConfig(resourceName string, deploymentId string) error {
+	if (resourceName == "" && deploymentId != "") || (resourceName != "" && deploymentId == "") {
```

Comment:
Could this just be simplified to:
```go
if (resourceName == "" || deploymentId == "")
```

Reply:
@parkerduckworth you're right about this change, I will fix it in all of the modules. Thanks!

Reply:
@parkerduckworth actually it can't be changed to this condition bc then it throws an error when both are empty